lib/suse/connect/client.rb

```diff
@@ -60,8 +66,9 @@ def deregister!
             deregister_product(product)
           end
           @api.deregister(system_auth)
+          log.info 'Cleaning up ...'
           System.cleanup!
-          log.info 'Successfully deregistered system.'
+          log.info "Successfully deregistered system.\n".red.bold
```

Comment:
IMO red is for errors, seeing "successfully" in red gives me cognitive dissonance. 

Also see all the error cases for register/deregister that are handled in SUSE::Connect::Cli::execute! - these could use red.